<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Прецессия и нутация гироскопа</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/Res/Logos/icon.ico</normaloff>:/Res/Logos/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontal" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>800</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="scene" native="true">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>840</width>
         <height>800</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="interface_2" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>460</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>460</width>
         <height>1600000</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QWidget" name="buttons" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>1600000</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QPushButton" name="start">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>43</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Calibri</family>
               <pointsize>17</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Запуск</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pause">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>43</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Calibri</family>
               <pointsize>17</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Пауза</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="stop">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>43</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Calibri</family>
               <pointsize>17</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Сброс</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="grid" native="true">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="18" column="0">
            <spacer name="verticalSpacer_8">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>5</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="6" column="0">
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>5</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="lengthL">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <family>Calibri</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="15" column="0">
            <widget class="QLabel" name="psi_dotL">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <family>Calibri</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="14" column="0">
            <spacer name="verticalSpacer_7">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>5</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="16" column="0">
            <widget class="QSlider" name="psi_dotS">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimum">
              <number>500</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QSlider" name="radiusS">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimum">
              <number>5</number>
             </property>
             <property name="maximum">
              <number>8</number>
             </property>
             <property name="value">
              <number>8</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="20" column="0">
            <widget class="QSlider" name="phi_dotS">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimum">
              <number>-30</number>
             </property>
             <property name="maximum">
              <number>30</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="22" column="0">
            <spacer name="verticalSpacer_9">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>5</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="radiusL">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <family>Calibri</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="massL">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <family>Calibri</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="24" column="0">
            <widget class="QSlider" name="thetaS">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimum">
              <number>785</number>
             </property>
             <property name="maximum">
              <number>1570</number>
             </property>
             <property name="value">
              <number>1570</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QSlider" name="massS">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>20</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <spacer name="verticalSpacer_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>5</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="23" column="0">
            <widget class="QLabel" name="thetaL">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <family>Calibri</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="19" column="0">
            <widget class="QLabel" name="phi_dotL">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <family>Calibri</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QSlider" name="lengthS">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimum">
              <number>15</number>
             </property>
             <property name="maximum">
              <number>20</number>
             </property>
             <property name="value">
              <number>20</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>107</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="timer_2" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QWidget" name="horizontalWidget" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="timerL">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16000</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>20</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="horizontalWidget_4" native="true">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>49</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="timerStart">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>43</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>17</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Запуск</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="timerPause">
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>43</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>17</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Пауза</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="timerStop">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>43</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>17</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Сброс</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>400</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>122</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="res.qrc">:/Res/Logos/NSTU.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu">
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Графики</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Графики энергии</string>
     </property>
     <addaction name="action_5"/>
     <addaction name="action_6"/>
     <addaction name="action_7"/>
    </widget>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="menu_4"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>О программе</string>
    </property>
    <addaction name="action_8"/>
    <addaction name="action_9"/>
    <addaction name="action_10"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>График зависимости θ от φ</string>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>График зависимости θ от времени</string>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Кинетическая энергия</string>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Потенциальная энергия</string>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Полная энергия</string>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Руководство пользователя</string>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Справка</string>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>Разработчики</string>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
